src/AcmTable/AcmTable.tsx

```diff
@@ -154,12 +154,15 @@ export function AcmTablePaginationContextProvider(props: { children: ReactNode;
     return <AcmTablePaginationContext.Provider value={paginationContext}>{children}</AcmTablePaginationContext.Provider>
 }
 
-export interface AcmTableProps<T> {
+export interface AcmTableProps<T, ST = void> {
     plural: string
     items?: T[]
+    subItems?: ST[]
     initialSelectedItems?: T[]
     columns: IAcmTableColumn<T>[]
     keyFn: (item: T) => string
+    groupFn?: (item: T, subItems: ST[]) => ST[]
+    subItemCellTransform?: (subItems: ST[]) => void
```

Comment:
Need a better name; but need a way to render a different component into the expanded row